pyproject.toml

```diff
@@ -0,0 +1,128 @@
+[tool.poetry]
+name = "cent"
+version = "5.0.0b1"
```

Comment:
Can we make version dynamic with Poetry? To not sync here and in __meta__

Reply:
You can research, I didn't found anything about this

Reply:
Let's remove from `__meta__` then? The idea is to have it in one place